src/Orckestra.Composer.Website/UI.Package/Typescript/Composer.Product/ProductDetail/ProductDetailController.ts

```diff
@@ -316,6 +318,71 @@ module Orckestra.Composer {
             })
         }
 
+        protected initKvaSelectVueComponent() {
+            let elId = 'vueKvaList';
+            let el = document.getElementById(elId);
+            if(!el) return;
+            let self: ProductDetailController = this;
+            $('[data-toggle="popover"]').popover({
+                placement:"top"
+            });
+            new Vue({
+                el: `#${elId}`,
+                data: {
+                    Product: self.context.viewModel,
+                    Variant: self.context.viewModel.selectedVariantId,
+                    Values: self.context.viewModel.keyVariantAttributeItems,
+                    Loading: false,
+                    dataUpdatedTracker: 1
+                },
+                computed: {
+                    KvaAttributeItems() {
+                        return this.Values;
+                    }
+                },
+                methods: {
+                    HasImage(value:any) {
```

Comment:
is it used? Does value has Imageurl ?

Reply:
value has ImageUrl but it is not used, removing